kustomization.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
+# Controller
+- name: objectstorage-controller
+  newName: quay.io/containerobjectstorage/objectstorage-controller
+  newTag: latest
+# Sidecar
+- name: sample-driver
+  newName: quay.io/containerobjectstorage/sample-driver
+  newTag: latest
+- name: object-storage-sidecar
+  newName: quay.io/containerobjectstorage/object-storage-sidecar
+  newTag: latest
+
+resources:
+# CRDs
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccessclasses.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccesses.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccessrequests.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketclasses.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketrequests.yaml
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_buckets.yaml
+# Controller
+- manifests/ns.yaml
```

Comment:
Generally, a `kustomization` should not deploy a namespace. It's up to the administrator to choose in which NS it gets deployed, and provision the NS.

Reply:
Captured in https://github.com/kubernetes-sigs/container-object-storage-interface-controller/issues/22